api/nakadi-event-bus-api.yaml

```diff
@@ -1024,6 +1024,26 @@ definitions:
         description: |
           Provides ability to set internal Nakadi parameters.
 
+      write_scopes:
+        type: array
+        items:
+          type: string
+        description: |
+          Provides ability to set oauth2 scopes to be able to publish an Event. The event publisher is cheked for
```

Comment:
I think it is good enough! My point was to describe it as wide as possible to cut additional questions in the chat room.
I tweaked it a bit:
This field is used to authorise clients for publishing to this event type. Nakadi authorises publishers whose token contains at least one of the scopes in this list. Publishers in their turn has to request scopes from event type owner.